package.json

```diff
@@ -4,29 +4,10 @@
   "description": "A tab component with drag and drop functionality, add and close",
   "main": "dist/bundle.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --u",
```

Comment:
Please remove the `--u` option. The updateSnapshot needs to used with consciousness.